### PR TITLE
Keygen

### PR DESCRIPTION
**Pull request by wmcrobertsq**
1. For this pull request, besides reviewing the code, attempt to exercise the code by following the readme's with the set of Qrypt Security libraries that will be sent separately.
2. Be careful with token setup. For now, the KeyGenDistributed Quickstart will use a staging token and the KeyGenLocal Quickstart will use a prod token.

### CODE REVIEW

File: KeyGenDistributed/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+- A Qrypt Account. [Create an account for free](https://portal.qrypt.com/).
+- Install [Visual Studio Code](https://code.visualstudio.com/).
+- Install [CMake](https://cmake.org/).
+- (Windows) Install git bash - comes with typical git install.
+
+## Setup
+1. Clone the repo containing this quickstart to a local folder on a Linux, Mac or Windows platform.
+1. Retrieve a token from the [Qrypt Portal](https://portal.qrypt.com/) and create an environment variable **QRYPT_TOKEN** for it.
+1. Download the Qrypt Security SDK for your platform.
+1. Create a lib folder and extract the Qrypt Security SDK into it.
```
Comment (alison-qrypt) [inline KeyGenDistributed/README.md:15]: Typo on the numbered list: should be "1.", "2.", "3.", "4."

Comment (alison-qrypt) [reply KeyGenDistributed/README.md:15]: If possible, you could consider to add actual linux commands sample for setup. For example,
```
git clone xxxx
curl -k xxxxxx   <------ the command to retrieve token
wget xxxxx <------ download the qrypt sdk
tar -xvzf qrypt-security-0.4.3-linux.tgz
cd qrypt-security-0.4.3-linux
```

Comment (wmcrobertsq) [reply KeyGenDistributed/README.md:15]: Ha! That's markdown syntax. The generated output will be automatically numbered correctly. In vs code, you can right click on tab for a markdown file and select "open preview" to see the generated output.

Comment (alison-qrypt) [reply KeyGenDistributed/README.md:15]: Oh I see. That's good to know. The linux command sample can be done in separate commit later.

File: KeyGenDistributed/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+Upon a successful build, the KeyGenDistributed console application can be found in the following folder:
+
+*Linux/Mac - KeyGenDistributed/build*
+
+*Windows - KeyGenDistributed/build/Debug* 
+
+Enter the following command for a complete set of build options:
+
+    ./build.sh --help
+
```
Comment (alison-qrypt) [inline KeyGenDistributed/README.md:41]: Suggestions:

- Make description concise and look like bullet points
- Display the current folder location at the command prompt sample

Taking Build section as example:
```
## Build

To change to the KeyGenDistributed folder:
    vm@vm:~$ cd KeyGenDistributed

To make a (debug) build:
    vm@vm:~/KeyGenDistributed$ ./build.sh --build_type=Debug

To find the build folder (if it built successfully):
    For linux:
        vm@vm:~/KeyGenDistributed$ ls -d build
    For windows:
        vm@vm:~/KeyGenDistributed$ ls -d build/Debug/

To see more build options:
    vm@vm:~/KeyGenDistributed$ ./build.sh --help
```

**Review by alison-qrypt — APPROVED** [review]

**Pull request merged**